yt/data_objects/tests/test_slice.py

```diff
@@ -39,40 +39,38 @@ def test_slice(pf):
         uc = [np.unique(c) for c in coords]
         slc_pos = 0.5
         # Some simple slice tests with single grids
-        for ax, an in enumerate("xyz"):
+        for ax in range(3):
             xax = ds.coordinates.x_axis[ax]
             yax = ds.coordinates.y_axis[ax]
-            for wf in ["density", None]:
```

Comment:
oh right slices don't have weight fields ... :)

Reply:
I guess not ? here I think the loop was just doing twice the same exact stuff.